nomad-core/tests/signer_integration.rs

```diff
@@ -0,0 +1,12 @@
+use ethers_signers::Signer;
+use nomad_core::Signers;
+use nomad_xyz_configuration::{agent::SignerConf, FromEnv};
+
+#[tokio::test]
+async fn signer_auths() {
+    if let Some(signer_conf) = SignerConf::from_env("TEST_KMS") {
```

Comment:
Changes to `from_env` here: https://github.com/nomad-xyz/rust/pull/171

Reply:
ooh, can we fix that to take a `&[&str]` with prefixes in precedence order?

Reply:
Replacing/simplifying with this interface https://github.com/nomad-xyz/rust/pull/180